doc/faq.rst

```diff
@@ -65,28 +65,20 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
+What are the inclusion criteria for algorithms ?
+-----------------------------------------------------
+As a rule we only add well-established algorithms. The minimal criterion is
 3 years since publications, 200+ citations and wide use and usefullness. A
 technique that provides a clear-cut improvement (e.g. an enhanced  data
-structure or efficient approximation) on a widely-used method will also be
-considered for inclusion.
-Your implementation doesn't need to be in scikit-learn to be used together
+structure or efficient approximation or both) on a widely-used method will also be considered for inclusion.
+Some exceptions may exist when an algorithm is not included  even if it does satisfy the above points. These may occur where constraints such as API's current ability to hold the algorithm, the amount of help needed to solve the future bugs and the hardness it creates for the current users' are not justified. 
```

Comment:
In general we only accept algorithms that fit well within the current API of scikit-learn, that is a ``fit``, ``predict``/``transform`` interface.